content/w/vic-golden.md

```diff
@@ -79,13 +79,14 @@ After [Golden Road Finals](@/e/mzw/2021-08-14-mzw-project-8-golden-road-finals.m
 He debuted at their [third Underground show](@/e/ptw/2022-03-27-ptw-underground-3.md) losing a few Singles Matches in a row.
 Along with a new ringname - Vic Golden - he also adopted a new gimmick of a flamboyant "Golden Boy", obsessed with jewelry and prestige.
 At the [sixth Underground](@/e/ptw/2022-06-26-ptw-underground-6.md), Vic and [Gabriel Queen](@/w/gabriel-queen.md) formed a tag team called Pure Gold.
-They feuded with PAKA and other teams, and were a strong candidate to win the Tag Team tournament, defeating such candidates as FoxJoy - [Justin Joy](@/w/justin-joy.md) and [Axel Fox](@/w/axel-fox.md).
+They feuded with PAKA and other teams, and were a strong candidate to win the Tag Team tournament, defeating such candidates as FoxJoy - [Justin Joy](@/w/justin-joy.md) and [Axel Fox](@/w/axel-fox.md). 
 
 Their final match as a team for PTW was during [PTW #4: The Mystery](@/e/ptw/2023-06-25-ptw-4-mystery.md), in the tournament final Tables Match for the newly created [PTW Tag Team Championship](@/c/ptw-tag-team-championship.md), which they however lost to PAKA.
-Later, they briefly joined forces with Legia Łysych ([Olgierd](@/w/olgierd.md) and [Marco Hammers](@/w/marco-hammers.md)), feuding against PAKA.
+Later, they briefly joined forces with Legia Łysych ([Olgierd](@/w/olgierd.md) and [Marco Hammers](@/w/marco-hammers.md)), feuding against PAKA. Their failure to get rid of PAKA and shifting the blame away from them later transitioned into a feud between Pure Gold and Legia themselves, with the pay-off only happening nearly a year later in [PpW](@/o/ppw.md).
+
 Golden's last match in PTW took place at [PTW x RYUCON 2](@/e/ptw/2023-07-16-ptw-x-ryucon.md) where he was defeated by [Diana Strong](@/w/diana-strong.md).
 
-In 2023, Pure Gold made appearances outside Poland, for Austrian Rings of Europe and Danish Nordic Elite Wrestling.
+In 2023, Pure Gold made an appearances outside of Poland, for Austrian Rings of Europe and Danish Nordic Elite Wrestling.
```

Comment:
```suggestion
In 2023, Pure Gold made appearances outside of Poland, for Austrian Rings of Europe and Danish Nordic Elite Wrestling.
```

Althouth both _outside_ and _outside of_ are correct, the latter is an Americanism.